Sources/SwifterSwift/SwiftStdlib/RawRepresentableExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RawRepresentableExtensions.swift
+//  SwifterSwift
+//
+//  Created by 方林威 on 2020/7/1.
+//  Copyright © 2020 SwifterSwift
+//
+
+// MARK: - Initializers
+public extension RawRepresentable {
+    
+    /// SwifterSwift: Initializes an  enumeration type. If `rawValue` was malformed, returns `nil`.
```

Comment:
I think its worth adding that the difference of this from the default native rawValue initializer is the argument is optional...

Reply:
yeah!